docs/_basic/authenticating_oauth.md

```diff
@@ -96,7 +96,7 @@ const app = new App({
       authVersion: 'v1', // default  is 'v2', 'v1' is used for classic slack apps
       metadata: 'some session data',
       installPath: '/slack/installApp',
-      redirectUriPath: '/slack/redirect',
+      redirectUriPath: 'http://example.com/slack/redirect',
```

Comment:
The XXXPath values are "path"s, not complete URLs. Thus, we are unable to reuse the parameter for `redirect_uri` in authorize URLs.

I feel that we should add `redirectUri` at the top-level, not under `installerOptions`. The reason why I think so is that we have `clientId`, `clientSecret`, and `scopes` there. The `redirectUri` should be in the same category as you set the value in the Slack app configuration page and it's not specific to an installer.

@stevengill Let me know if you have a different thoughts on this 👋 

See also: 
* https://github.com/slackapi/bolt-python/blob/main/docs/_basic/authenticating_oauth.md
* https://slack.dev/java-slack-sdk/guides/app-distribution

<img width="500" src="https://user-images.githubusercontent.com/19658/133532459-03522cab-b094-4d19-a871-f6aafb8e512c.png">
<img width="500" src="https://user-images.githubusercontent.com/19658/133532467-9b5fa3f8-3346-42b1-84e2-c6b1725edcf5.png">

Reply:
Oh interesting! I didn't know there was both redirect url and a redirect uri path which you can supply in the python sdk. If I'm reading the documentation correctly, is it our recommendation that users just skip adding this on the App client level and configure directly in the Slack app config? 

It seems like on the Bolt-JS the way it was implemented just totally sidesteps allowing users to actually supply a redirectUri and relies on them using the Slack app config instead. But it makes it seem as though they can supply a value which is incorrect. 

To confirm, you're suggesting we keep redirectUriPath as it is and create a new redirectUri value at the top-level?

Reply:
Sounds good to me! Let's add a new top level `redirectUri` property that contains the full url

Reply:
>To confirm, you're suggesting we keep redirectUriPath as it is and create a new redirectUri value at the top-level?

Yes, your understanding is correct.

Reply:
Thanks for weighing in on this! I have included a top-level `redirectUri` and some checks to make sure both `redirectUri` and `redirecUriPath` are supplied in my latest changes.